src/Bundle.ts

```diff
@@ -107,6 +189,29 @@ export default class Bundle {
 						this.graph.legacy,
 						options.freeze !== false
 					);
+
+					module.dynamicImportResolutions.forEach((replacement, index) => {
```

Comment:
The arrow function here might be a good candidate for an extracted private method